mailpoet/views/welcome_wizard.html

```diff
@@ -26,22 +26,13 @@
 <%= localize({
 'welcomeWizardLetsStartTitle': __('Welcome! Let’s get you started on the right foot.'),
 'welcomeWizardSenderText': __('Who is the sender of the emails you’ll be creating with MailPoet?'),
-'welcomeWizardUsageTrackingStepTitle': __('Help MailPoet improve with anonymous usage tracking.'),
-'welcomeWizardUsageTrackingStepSubTitle': __('Data we don’t gather:'),
-'welcomeWizardUsageTrackingStepTrackingLabel': __('Do you want to share anonymous data and help us improve the plugin? We appreciate your help!'),
-'welcomeWizardUsageTrackingStepTrackingLabelNote': __('This requires loading 3rd-party libraries enabled.'),
+'welcomeWizardUsageTrackingStepTitle': __('Privacy and data sharing'),
+'welcomeWizardUsageTrackingStepTrackingLabel': __('Help improve MailPoet'),
+'welcomeWizardUsageTrackingStepTrackingLabelNote': __('Get improved features and fixes faster by sharing with us [link]non-sensitive data about how you use MailPoet[/link]. No personal data is tracked or stored.'),
 'welcomeWizardUsageTrackingStepTrackingLabelNoteNote': __('Note'),
-'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNote': __('This needs to be enabled if you want to be able to contact support.'),
-'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNoteNote': __('Important'),
-'welcomeWizardUsageTrackingStepLibs3rdPartyLabel': __('Do you want to enable loading 3rd-party libraries, like Google Fonts (used in Form Editor) and HelpScout (used for support)?'),
-'welcomeWizardUsageTrackingStepLibs3rdPartyLink': __('Read more about libraries we use.'),
-'welcomeWizardTrackingText': __('Gathering usage data allows us to make MailPoet better — the way you use MailPoet will be considered as we evaluate new features, judge the quality of an update, or determine if an improvement makes sense.'),
-'welcomeWizardTrackingList1': __('Any personal data'),
-'welcomeWizardTrackingList2': __('Email addresses'),
-'welcomeWizardTrackingList3': __('Login and passwords'),
-'welcomeWizardTrackingList4': __('Content of your emails'),
-'welcomeWizardTrackingList5': __('Open and click rates'),
-'welcomeWizardTrackingLink': __('Read more about what we collect.'),
+'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNote': __('If enabled, we may load the Google Fonts library and [link]other 3rd-party libraries we use[/link].'),
+'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNoteNote': __('Note'),
+'welcomeWizardUsageTrackingStepLibs3rdPartyLabel': __('Enable better-looking Google fonts in emails and show contextual help articles in MailPoet?'),
```

Comment:
Hi @lysyjan

For this part, with regards to Translations, would it be better to create a new text string, or is it alright the way it is? 

I edited the contents of the text tag, and I'm wondering if it would show up in the Translation client as needing to be re-translated.

Reply:
@triple0t If I am not wrong, our command extracts strings from the `__()` function. So we don't need to change the keys. [Here](https://mailpoetp2.wordpress.com/team-handbook/engineering-process/plugins-translations/) is a P2 page about how translations work.

Reply:
Thanks @lysyjan. I will check it out.

> our command extracts strings from the __() function.

To be confirm, 
e.g 
**before**
```
'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNoteNote': __('Important'),
```

**after**
```
'welcomeWizardUsageTrackingStepLibs3rdPartyLabelNoteNote': __('Note'),
```

The Key is the same `welcomeWizardUsageTrackingStepLibs3rdPartyLabelNoteNote` but the text content is not.

Here, the command will extract out `'Note'` as the new text right?

Reply:
@triple0t Yes, this is exactly how I understand it.

Reply:
Awesome 👏, Thank you @lysyjan